graphql/schemabuilder/reflect.go

```diff
@@ -328,8 +355,8 @@ func makeStructParser(typ reflect.Type) (*argParser, graphql.Type, error) {
 	}, argType, nil
 }
 
-func makeSliceParser(typ reflect.Type) (*argParser, graphql.Type, error) {
-	inner, argType, err := makeArgParser(typ.Elem())
+func (sb *schemaBuilder) makeSliceParser(typ reflect.Type) (*argParser, graphql.Type, error) {
```

Comment:
in the future, I would make the change that make make***Parser all methods its own commit instead